### PR TITLE
[FDD] new histograms to have vertex and collision time information

### PR DESCRIPTION
**Pull request by arvindkhuntia**

### CODE REVIEW

**Comment by arvindkhuntia** [comment]
Hi @afurs, if possible please have a look.

File: Modules/FIT/FDD/src/RecPointsQcTask.cxx [hunk 1]
```diff
@@ -177,19 +195,32 @@ void RecPointsQcTask::monitorData(o2::framework::ProcessingContext& ctx)
...
+
     auto channels = recpoint.getBunchChannelData(chan);
     mHistBC->Fill(recpoint.getInteractionRecord().bc);
 
     if (isTCM) {
-      mHistBCTCM->Fill(recpoint.getInteractionRecord().bc);
+      int bc = recpoint.getInteractionRecord().bc;
+      bool vertexTrigger = triggersignals.getVertex();
+      double collisionTimeA = static_cast<Float_t>(recpoint.getCollisionTimeA() * 1.e-3); // time ps-->ns
+      double collisionTimeC = static_cast<Float_t>(recpoint.getCollisionTimeC() * 1.e-3); // time ps-->ns
```
[the remark this answers is not in the document]
Comment (arvindkhuntia) [reply Modules/FIT/FDD/src/RecPointsQcTask.cxx]: Good Point. Now changed to double type.

**Comment by arvindkhuntia** [comment]
Hi @Barthelemy Could you please merge this branch?

Many thanks.

**Comment by arvindkhuntia** [comment]
Hi @Barthelemy, Could you please merge this PR.

Many thanks.

**Comment by Barthelemy** [comment]
error unrelated, merging.

**Pull request merged**